server/indexers/gh.py

```diff
@@ -10,15 +10,24 @@
 from gh_pages import index as gh_pages
 from gh_issues import index as gh_issues
 
-gh_pool = urllib3.connection_from_url(settings.GITHUB.get('GH', {}).get('WEB'), maxsize=50, block=True)
+headers = {
+    'keep_alive': True,
+    'user_agent': 'cfpb-tiresias',
+}
+
 gh_api_client = Client(gh_settings.get('API'))
```

Comment:
It looks like all the universal client code is not being used (except for jira's process).  You should probably pull out all the unused instances.